src/node_nanomsg.cc

```diff
@@ -23,6 +23,11 @@ using v8::Object;
 using v8::String;
 using v8::Value;
 
+#define ret NanReturnValue
+#define utf8 v8::String::Utf8Value
```

Comment:
`nn_setsockopt` and `nn_getsockopt` take a (signed) int as their first argument.  You shouldn't be using a precise type for int, instead defer to the host's definition of an int.